src/pkg/backup/backup.go

```diff
@@ -37,7 +37,7 @@ type Backup struct {
 	Version int `json:"version"`
 
 	// Errors contains all errors aggregated during a backup operation.
-	Errors fault.ErrorsData `json:"errors"`
+	Errors fault.Errors `json:"errors"`
```

Comment:
this needs tested (manually or unit test) to ensure it still allows deserializing backups that had errors

Reply:
confirmed all green